### ISSUE TITLE
cython_function_or_method object does not support vectorcall

### ISSUE THREAD

**Issue by DavidLP** [SiLab-Bonn/pylandau#17]
Via E-Mail the following issue was reported:

With python 3.8, the pip install way works smoothly. 

But if I try to run any of the examples I have the following issue raised : 
```
File "fit.py", line 14, in <module>
    y = pylandau.langau(x, mpv, eta, sigma, A)
  File "pyLandau\cpp\pylandau.pyx", line 97, in pylandau.langau (pyLandau/cpp/pylandau.cpp:4339)
  File "pyLandau\cpp\pylandau.pyx", line 124, in pylandau._scale_to_mpv (pyLandau/cpp/pylandau.cpp:5379)
  File "/afs/cern.ch/user/l/***/miniconda2/envs/***/lib/python3.8/site-packages/scipy/optimize/optimize.py", line 543, in fmin
    res = _minimize_neldermead(func, x0, args, callback=callback, **opts)
  File "/afs/cern.ch/user/l/***/miniconda2/envs/***/lib/python3.8/site-packages/scipy/optimize/optimize.py", line 689, in _minimize_neldermead
    fsim[k] = func(sim[k])
  File "/afs/cern.ch/user/l/***/miniconda2/envs/***/lib/python3.8/site-packages/scipy/optimize/optimize.py", line 427, in function_wrapper
    return function(*(wrapper_args + args))
TypeError: 'cython_function_or_method' object does not support vectorcall
```

**Comment by DavidLP** [comment]
This looks like a bug in `scipy` to me and I guess has to be fixed there. A workaround is to use `python 3.7` instead of `python3.8` for now, since the vectorcall API was introduced in python3.8.

**Comment by ChristianBesp** [comment]
I think this is solved in #19 after recompiling with a recent cython version. Tests pass under python3.8 there.
I will close this and merge #19 soonish.
